arkane/mainTest.py

```diff
@@ -79,6 +80,31 @@ def test_arkane_examples(self):
             error_message += '{1} in {0}; '.format(example_name, example_type)
         self.assertFalse(self.failed, error_message)
 
+    def test_arkane_two_parameter_arrhenius_fit(self):
+        test_path = os.path.join(self.test_base_path, 'two_parameter_arrhenius_fit')
+        file_to_remove = ['output.py', 'chem.inp', 'supporting_information.csv']
+        for file in file_to_remove:
+            if os.path.exists(os.path.join(test_path, file)):
+                os.remove(os.path.join(test_path, file))
+        arkane = Arkane(input_file=os.path.join(test_path, 'input.py'), output_directory=test_path)
+        arkane.plot = False
+        arkane.save_rmg_libraries = False
+        logging.info('running two_parameter_arrhenius_fit test')
```

Comment:
no need to log this, nosetests does it for us

Reply:
ok